ec2/spark_ec2.py

```diff
@@ -872,9 +890,16 @@ def deploy_files(conn, root_dir, opts, master_nodes, slave_nodes, modules):
     if "." in opts.spark_version:
         # Pre-built Spark deploy
         spark_v = get_validate_spark_version(opts.spark_version, opts.spark_git_repo)
+        tachyon_v = get_tachyon_version(spark_v)
     else:
         # Spark-only custom deploy
         spark_v = "%s|%s" % (opts.spark_git_repo, opts.spark_version)
+        if opts.tachyon_version is None:
```

Comment:
What I meant is that we could fall back to some default tachyon version (say 0.6) if spark git hash is set but tachyon version is not set. We don't need to do force the user to pass this option then but if the user needs some specific tachyon version or git-hash they can set it.

This can be done by just setting a default value for that flag.